test/token-locking.js

```diff
@@ -68,8 +68,7 @@ contract("TokenLocking", accounts => {
       await checkErrorRevert(
         tokenLocking.deposit(token.address, usersTokens, {
           from: userAddress
-        }),
-        "token-locking-transfer-failed"
```

Comment:
The error message generated from `deposit` is actually empty, as a nested error is being generated and overrides the one we specified.

The expected error: https://github.com/JoinColony/colonyNetwork/blob/develop/contracts/TokenLocking.sol#L69

The actual error occurs at: https://github.com/dapphub/dappsys-monolithic/blob/002389d43cf54e8f0b919fee1fc364b20ebdf626/base.sol#L52

Not approved means the `_approvals[src][msg.sender]` value is 0, and the test attempts to subtract 6 from 0.

Reply:
Good spot @KevinLiLu 
Note the `require` failure in `DSMath` contract and any external libraries in general is out of scope for us

Reply:
I'm just going to remove the require part then? 
https://github.com/JoinColony/colonyNetwork/blob/e5de2fac169a29c90749fc438db17bd08642cba8/contracts/TokenLocking.sol#L69

`transferFrom` always returns true unless there is some error (in which it errors instead of returning false).
https://github.com/dapphub/dappsys-monolithic/blob/002389d43cf54e8f0b919fee1fc364b20ebdf626/base.sol#L47-L61

Reply:
yes sure

Reply:
Doesn't removing the require go against what we were advised in review we got done of the vesting contract?

Reply:
Ok I will leave it as is then. 

The current test case will not be able to check the `token-locking-transfer-failed` error message then... Do we need to create a new test case? This is the only place where we test the error message `token-locking-transfer-failed`.